topdown/crypto.go

```diff
@@ -177,23 +181,57 @@ func builtinCryptoX509ParseRSAPrivateKey(_ BuiltinContext, operands []*ast.Term,
 	}
 
 	// get the raw private key
-	rawKey, err := getRSAPrivateKeyFromString(string(input))
+	rawKey, err := getPrivateKeyFromString(string(input))
+	if err != nil {
+		return err
+	}
+
+	if len(rawKey.([]crypto.PrivateKey)) != 0 {
+
+		rsaPrivateKey, err := jwk.New(rawKey.([]crypto.PrivateKey)[0])
```

Comment:
suggestion: I think it's worth explaining here that if there are many keys found in the input, we'll use the first one only.